src/styles/blocks/variables.scss

```diff
@@ -0,0 +1,18 @@
+$days-of-mounth: 30;
+$start-day: fri;
```

Comment:
this values should be directly in modifiers, so I can change class in browser and see the result)